doc/src/manual/mathematical-operations.md

```diff
@@ -63,6 +63,22 @@ julia> false * Inf
 
 This is useful for preventing the propagation of `NaN` values in quantities that are known to be zero. See [Knuth (1992)](https://arxiv.org/abs/math/9205211) for motivation.
 
+## Boolean Operators
+
+The following [boolean operators](https://en.wikipedia.org/wiki/Boolean_algebra#Operations) are supported on Bool:
+
+| Expression | Name                                  |
+|:---------- |:--------------------------------------|
+| `!x`       | not                                   |
+| `x & y`    | and                                   |
+| `x \| y`   | or                                    |
+| `x ⊻ y`    | xor (exclusive or)                    |
```

Comment:
```suggestion
```

These are documented below and I'd rather push folks towards using `&&` and `||` for boolean logic as it has the precedence they likely want.

Reply:
Sure. Done.